src/overlays/effects/ovl_Effect_Ss_Sbn/z_eff_ss_sbn.h

```diff
@@ -4,7 +4,9 @@
 #include "global.h"
 
 typedef struct {
-    char unk_0[UNK_SIZE];
+    /* 0x00 */ Vec3f unk_00;
+    /* 0x0C */ CollisionPoly* unk_0C;
+    /* 0x10 */ f32 unk_10;
```

Comment:
Could you add the `char unk_14[UNK_SIZE]` to the end of this struct, unless you know these are the only members, in which case should fill out the size info.